packages/@vue/cli-ui-addon-webpack/src/components/BuildProgress.vue

```diff
@@ -166,7 +166,7 @@ export default {
       width 100%
       height @width
       border-radius 50%
-      background rgba(black, .1)
+      background $vue-ui-color-dark
```

Comment:
This changes the color in non-dark mode too.
Should be
```suggestion
      background rgba(black, .1)
      .vue-ui-dark-mode &
        background: $vue-ui-color-dark
```